src/playback-watcher.js

```diff
@@ -352,6 +460,8 @@ export default class PlaybackWatcher {
       this.logger_(`Stopped at ${currentTime}, setting timer for ${difference}, seeking ` +
         `to ${nextRange.start(0)}`);
 
+      this.cancelTimer_();
```

Comment:
Noticed that it's technically possible to have two gap skipper timers, although I would wager that it is unlikely.